aea/aea.py

```diff
@@ -184,6 +186,11 @@ def __init__(
 
         self._setup_loggers()
 
+    @classmethod
+    def get_build_dir(cls) -> str:
+        """Get agent build directory."""
+        return cls.DEFAULT_BUILD_DIR_NAME
```

Comment:
I don't see the point in defining this here.

Reply:
will discuss later